src/components/CellsContainer/helpers.js

```diff
@@ -0,0 +1,92 @@
+import { create } from 'd3';
+
+export function getCellSize(totalCellsCount) {
+  switch (true) {
+    case totalCellsCount < 100:
+      return 38;
+
+    case totalCellsCount < 500:
+      return 20;
+
+    case totalCellsCount < 1000:
+      return 18;
+
+    case totalCellsCount < 2000:
+      return 16;
+
+    default:
+      return 11;
+  }
+}
+
+export function getCellProps(index, itemsCount) {
+  const [lived, total] = itemsCount;
+  const color = index < lived ? 'IndianRed' : 'SteelBlue';
+
+  return {
+    color,
+    tooltipText: 'Number: ' + ++index
+  };
+}
+
+export function setAttributes(el, attrs) {
+  Object.keys(attrs).forEach((attr) => {
```

Comment:
```javascript
export function setAttributes(el, attrs = {}) {
  Object.entries(attrs).forEach(([arrt, value]) => el.attr(attr, value))
}
```
Or... 😏 
```javascript
export const setAttributes = (el, attrs = {}) =>
  Object.entries(attrs).forEach(attr => el.attr(...attr));
```
Somebody stop me 🤪 
```javascript
import {entries, each, spread} from 'lodash';

export const setAttributes = (el, attrs = {}) =>
  each(entries(attrs), spread(el.attr));
```

Reply:
ahaha cool!
That make sense)
I forgot about Object.entries (didn't use it before)
But lodash is great! I definitely need to meet him closer)

Reply:
Careful, it could be addictive! 😄